addons/web/static/src/core/datetime/datetime_picker_popover.js

```diff
@@ -23,6 +23,10 @@ export class DateTimePickerPopover extends Component {
 
     static template = "web.DateTimePickerPopover";
 
+    get isDateTimeRange() {
+        return this.props.pickerProps.type === 'datetime' || Array.isArray(this.props.pickerProps.value);
```

Comment:
this line is too long. Consider using the web tooling (`./addons/web/tooling/enable.sh`). It activates a pre-commit hoot that lints/prettifies the code.

Reply:
Changes has been done.
Thank You!